feature/gribi/ate_tests/hierarchical_weight_resolution_test/hierarchical_weight_resolution_test.go

```diff
@@ -286,7 +284,6 @@ func (a *attributes) configureNetworkInstance(t *testing.T, d *ondatra.DUTDevice
 		fptest.LogQuery(t, "NI", dni.Config(), gnmi.GetConfig(t, d, dni.Config()))
 	} else {
```

Comment:
could you please guard this code block with ExplicitInterfaceInDefaultVRF deviation.